R/jss_article.R

```diff
@@ -39,6 +39,16 @@ jss_article <- function(..., keep_tex = TRUE, citation_package = 'natbib') {
     )
   }
 
+  pre <- base$pre_processor
+  base$pre_processor <- function(metadata, input_file, runtime,
```

Comment:
I'd probably add a `pandoc_args` argument to `jss_article()`, and pass `--lua-filter` to it, instead of using a pre-processor.

Reply:
Oh yes great idea! Thank you !